examples/AdminClient/Program.cs

```diff
@@ -583,27 +583,49 @@ static async Task ListConsumerGroupsAsync(string bootstrapServers, string[] comm
         {
             var timeout = TimeSpan.FromSeconds(30);
             var statesList = new List<ConsumerGroupState>();
-            try
+            var groupTypesList = new List<ConsumerGroupType>();
+            var isType = false;
+            var isState = false;
+            for (int i = 0; i < commandArgs.Length; i++)
             {
-                if (commandArgs.Length > 0)
+                if (commandArgs[i] == "-states")
                 {
-                    timeout = TimeSpan.FromSeconds(Int32.Parse(commandArgs[0]));
+                    if (isState)
+                    {
+                        Console.WriteLine("usage: .. <bootstrapServers> list-consumer-groups [-states <match_state_1> <match_state_2> ... <match_state_N>] [-types <group_type_1> .. <group_type_M>]");
```

Comment:
Improve the error message, state that this was a Duplicate states argument. Same for types

Reply:
No the duplicate state is Invalid args but still the example will run and yield a error like Invalid args.
This is basically error with exit code so user has put in wrong input format for the example.
like -state_cnt=2 state_1 -group_type_cnt=..
The above will error because only 1 state was given.

Reply:
Please suggest something if you feel it is still not sufficient.